### DISCUSSION TITLE
Chat for moderators/community?

### DISCUSSION THREAD

**Discussion by jedieaston** [microsoft/winget-pkgs#25866] [Ideas] [answered]
Now that we're adding moderators, does anyone have good ideas about some kind of chat or something outside of GitHub Issues/PRs? Coordinating moderation at the bottom of PRs has made my GitHub notifications a mess (because I can often not tell at a glance if people are discussing something I need to know or if it's just @msftbot letting me know it saw I approved a PR). When there's a major refactor (like last month when I changed all of the Visual Studio manifests again and had to launch 130 PRs into the queue), I'd like to tell someone so that there isn't confusion, but I don't know the best way to do it. When manifest 1.1 launches we'll probably want somewhere to organize converting the manifests that need it.

Discord is an option (I'd be happy to set that up, and it's not too bad to moderate), another option is [gitter](https://gitter.im/Microsoft/vscode), which is used by the VSCode team to interact with their community members. Or we can just be more diligent about moving stuff into discussions (although sometimes notifications don't fire with that intuitively, like if someone replies under a parent comment you haven't commented on). Thoughts?

Tagging people I would like to give an opinion:
@denelon, @ItzLevvie, @OfficialEsco, @ImJoakim, @KaranKad, and @quhxl.

**Comment by ImJoakim** [comment]
For sure, having a place where we can quickly discuss and talk about stuff would be great if you ask me. I can use anything, it doesn't matter that much to me. I'm assuming most people have and are familiar with Discord, so that could be used. But I'm down to use anything (with the assumption that I get accepted as a moderator).

**Comment by OfficialEsco** [comment]
I am definitively voting for Gitter since i despise Discord.
Gitter is Open Source, have GitHub Integration and Matrix/Element integration (Huge plus since privacy minded people can join) and you can see Issues(?) in the Activity panel, and Microsoft already have a bunch of Rooms.

Reference: the TypeScript room https://gitter.im/Microsoft/TypeScript?source=orgpage#

**Reply by jedieaston** [reply]
Is there a way in Gitter to have multiple channels for one room similar to Discord/Slack/Teams? (I can't find the button, although the two Gitter rooms I'm in I don't have any special rights so maybe it is hidden)

**Comment by denelon** [comment, marked as answer]
I'm totally new to Gitter, but I created a channel.

https://gitter.im/Microsoft/winget-pkgs
